app/tailwind.config.js

```diff
@@ -1,13 +1,12 @@
 module.exports = {
   content: ["./pages/**/*.{ts,tsx}", "./config/**/*.{ts,tsx}", "./components/**/*.{ts,tsx}", "./pages/_app.tsx"],
   options: {
-    safelist: ["landingPageBackground", "mobileLandingPageBackground"],
+    safelist: ["passportLanding", "mobilePassportLanding"],
```

Comment:
Do these not refer to the `extend.bakgroundImage` props below? If not, what are they for, I don't see `mobilePassportLanding` anywhere else in the diff?

Reply:
Ah, I meant to take that out of config file since I ended up not using a different svg for mobile. I'll remove it from the config and push up the changes!

Reply:
I ended up removing the `safeList` option all together and the background since this design is just an image, not an entire background 🚀